GPCS4/SceModules/SceFiber/sce_fiber.h

```diff
@@ -19,29 +19,44 @@ extern const SCE_EXPORT_MODULE g_ExpModuleSceFiber;
 // The codebase is generated using GenerateCode.py
 // You may need to modify the code manually to fit development needs
 
+typedef void PS4API (*SceFiberEntry)(uint64_t argOnInitialize, uint64_t argOnRun) __attribute__((noreturn));
```

Comment:
these's a `PS4NORETURN` for `__attribute__((noreturn))`
see:
https://github.com/Inori/GPCS4/blob/master/GPCS4/Common/GPCS4Decoration.h